be/src/exec/pipeline/sink/table_function_table_sink_operator.cpp

```diff
@@ -21,7 +21,7 @@
 namespace starrocks::pipeline {
 
 // TODO(letian-jiang): optimize
-StatusOr<std::string> column_to_string(const TypeDescriptor& type_desc, const ColumnPtr& column) {
+static StatusOr<std::string> column_to_string(const TypeDescriptor& type_desc, const ColumnPtr& column) {
```

Comment:
put this function into some header files, and no need to make it static since it's not a class member function.